packages/devextreme/js/core/component.js

```diff
@@ -82,6 +99,32 @@ export const Component = Class.inherit({
         this._disposingCallbacks = _disposingCallbacks || Callbacks();
         this.postponedOperations = new PostponedOperations();
         this._createOptions(options);
+
+        if(!licenseVerified) {
+            licenseVerified = true;
+            const licenseToken = Config().license;
+            if(licenseToken) {
+                const license = parseToken(licenseToken);
+
+                if(license.kind === 'corrupted') {
+                    if(license.error === 'version') {
+                        errors.log(invalidVersionLicenseMessage);
+                    } else {
+                        errors.log(invalidFormatLicenseMessage);
+                    }
```

Comment:
`version` in this context means token format version, not devextreme version
```suggestion
                    errors.log(invalidFormatLicenseMessage);
```